addons/refuel/stringtable.xml

```diff
@@ -479,7 +479,11 @@
             <Portuguese>Comprimento da Mangueira de Combustível</Portuguese>
             <French>Longueur du tuyau</French>
             <Czech>Délka hadice na palivo</Czech>
-            <Turkish>Yakıt poması uzunluğu</Turkish>
+            <Turkish>Yakıt hortumu uzunluğu</Turkish>
+        </Key>
+        <Key ID="STR_ACE_Refuel_RefuelSettings_progressDuration_DisplayName">
+            <English>Progress bar duration</English>
```

Comment:
"Pump Interaction Time" might sound a little more natural, but it's fine as is

Reply:
It effects all overall pump, hose, vehicle interaction time so I wanted to make it obvious it' all interactions times. Could maybe call it just "Interaction Time", and then add a hint that explains it better?